modules/apps/transfer/types/transfer_authz.go

```diff
@@ -0,0 +1,105 @@
+package types
```

Comment:
small nit here as well, but can we name it transfer_authorization.go so it's similar to [this](https://github.com/cosmos/cosmos-sdk/blob/3bb27795742dab2451b232bab02b82566d1a0192/x/bank/types/send_authorization.go)?